lms/djangoapps/teams/static/teams/templates/teams_tab.underscore

```diff
@@ -7,6 +7,7 @@
     </div>
 </div>
 <div class="teams-header"></div>
+<div class="teams-instructor-tools-bar"></div>
```

Comment:
@andy-armstrong this is the crux of the architectural issue I mentioned in standup, when you have a minute